libmscore/textbase.cpp

```diff
@@ -519,8 +519,35 @@ void TextFragment::draw(QPainter* p, const TextBase* t) const
       {
       QFont f(font(t));
       f.setPointSizeF(f.pointSizeF() * MScore::pixelRatio);
+#if defined Q_OS_WIN || defined Q_OS_LINUX
+      if (f.bold()) {
+            // workaround for https://musescore.org/en/node/284218
+            // and https://musescore.org/en/node/281601
+            // only needed for certain artificially emboldened fonts
+            // see https://musescore.org/en/node/281601#comment-900261
+            // in Qt 5.12.x this workaround should be no more necessary if
+            // env variable QT_MAX_CACHED_GLYPH_SIZE is set to 1
+            p->save();
+            qreal mm = p->worldTransform().m11();
+            qreal dx = p->worldTransform().dx();
+            qreal dy = p->worldTransform().dy();
+            qreal factor = 1 / mm; // correction factor for bold text drawing
+            p->setMatrix(QMatrix(mm*factor,0.0,0.0,mm*factor,dx,dy));
```

Comment:
please a spaces after the commas

Reply:
Ok. Thank you Jojo-bot 😉 
Will update the PR in a couple of minutes.